day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+(1..100).each {|num|
```

Comment:
Nice job on Fizzbuzz Scott! This is  about as simple and elegant of a solution as one can hope for! I encourage you to attempt the other challenges from Day-7 under a time constraint of a few hours if you have a chance this weekend! This is great practice for what you will face in Mod-1, and it will keep honing those ruby skills! Great job on the capstone and in Mod-0!